plugins/backend/catalog-backend-module-okta/src/providers/filters/filterOperators.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2022 Larder Software Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { FilterOperator } from '../../types';
+
+export const filterOperators: Record<
+  FilterOperator,
+  (lhs: any, rhs: any) => boolean
+> = {
+  equals: (lhs: any, rhs: any) => {
```

Comment:
What is lhs, rhs ?

Reply:
left had side and right hand side. When referring to operators, you can call each side of the operator by the side its on.

e.g.

`1 < 2`

In this case, the left hand side is 1, and right hand side is 2

I am happy to give them different names if you have better names?

Reply:
@kissmikijr does this answer your question?